internal/characters/keqing/cons.go

```diff
@@ -42,6 +43,10 @@ func (c *char) c2() {
 
 func (c *char) c4() {
 	cb := func(args ...interface{}) bool {
+		if _, ok := args[0].(*gadget.Gadget); ok {
```

Comment:
why this check when there is one in `cbNoGadget`?

Reply:
I mixed up some naming things while working on Keqing. I'll correct it.

Reply:
It's fixed